packages/devtools_app/test/treemap_test.dart

```diff
@@ -148,6 +148,87 @@ void main() {
         skip: kIsWeb || !Platform.isMacOS,
       );
 
+      testWidgetsWithWindowSize(
+        'builds treemap with expected data after zooming in',
+        windowSize,
+        (WidgetTester tester) async {
+          const treemapKey = Key('Treemap');
+          await pumpTreemapWidget(tester, treemapKey);
+
+          var text = 'package:flutter [2.99 MB]';
+          expect(find.text(text), findsOneWidget);
+          await tester.tap(find.text(text));
+          await tester.pumpAndSettle();
+
+          await pumpTreemapWidget(tester, treemapKey);
+
+          text = 'dart:ui [230 KB]';
+          expect(find.text(text), findsNothing);
+
+          await expectLater(
+            find.byKey(treemapKey),
+            matchesGoldenFile('goldens/treemap_v8_zoom.png'),
+          );
+          // Await delay for golden comparison.
+          await tester.pumpAndSettle(const Duration(seconds: 2));
+        },
+        skip: kIsWeb || !Platform.isMacOS,
+      );
+    },
+  );
+  group(
+    'Treemap from the difference between two v8 snapshots',
```

Comment:
I'm not seeing how this is using different data from the previous testing group. What about this is diff tree specific?